argyrogounari/assignment1/Question1.java

```diff
@@ -0,0 +1,85 @@
+import java.util.Scanner;
+import java.util.*;
+
+public class Question1 {
+
+	public static void main(String[] args) {
+		
+		Scanner reader = new Scanner(System.in);  
+		System.out.println("Enter your strings: ");
+		String sentence1 = reader.nextLine();
+		String sentence2 = reader.nextLine();
+		
+		reader.close(); 
+		
+		if (anagram(sentence1,sentence2)) {
+			System.out.printf("'%s' and '%s' are anagrams",sentence1, sentence2);
+		} else {
+			System.out.printf("'%s' and '%s' are NOT anagrams",sentence1, sentence2);
+		}
+
+	}
+	
+	public static boolean anagram(String sentence1, String sentence2){
```

Comment:
This function returns a boolean; so the name would benefit from being a statement like isAnagram.